### ISSUE TITLE
❌ ♿ Accordion uses aria-selected="true"

### ISSUE THREAD

**Issue by dgullerud-usc** [elementor/elementor#36152]
### Description

In the accordion widget, aria-selected="true" is being used. For an accordion, aria-expanded and aria-controls should be used instead. 

### Steps to reproduce

https://lgbtqplus.usc.edu/involvement/student-employment/

### Expected behavior

aria-expanded and aria-controls should be used instead.

### Elementor System Info

```txt
Elementor Version 4.1.1 
Theme: Neve
```

### Agreement

- [ ] I confirm I have read and followed all the guidelines and instructions outlined in the Elementor Bug Report form.
- [x] I agree that my issue may be closed without further action if it doesn't meet all the requirements outlined in the Elementor Bug Report form.

**Comment by nicholaszein** [comment]
Hi. This issue doesn't follow the [Contribution Guidelines for Reporting Bugs](https://elemn.to/gh-contributing-bug-reports), also outlined in the [Bug Report Form](https://elemn.to/gh-new-bug-report). It's missing key information that is required to properly investigate the issue. 

For this reason I'm closing it. 

If you still think you found a bug. Please submit a new issue with all the required information in English. 

Regards
